### PR TITLE
C renames in the code: TSS => ShorePayment

### PR DESCRIPTION
**Pull request by cs**
Renames the top-level namespace. I want to do this, before we merge our branch in Core into `master`. PR for the Core project to integrate the rename coming once this is merged...

ping @ceigel

### CODE REVIEW

**Comment by cs** [comment]
ping @sadtuna

File: lib/payment/connector.rb [hunk 1]
```diff
@@ -5,10 +5,12 @@
...
 require_relative 'response_handlers'
 
-module TSS
-  # Utility class encapsulating synchronous communication with TSS.
+module ShorePayment
+  # Utility class encapsulating synchronous communication with Shore's Payment
+  #   Service.
   class Connector
     include ResponsesHandlers
+
```
Comment (ceigel) [inline lib/payment/connector.rb]: useless empty line

**Comment by ceigel** [comment]
:+1:

**Comment by sadtuna** [comment]
:+1:

**Pull request merged**